File: 3_list/characters_to_string.rb
# Convert a list of characters into a string
# Input ['a', 'b', 'c', 'd']
# Output abcd

def character_to_string
  print "Number of characters: "
  number_characters = gets.chomp.to_i

  characters = characters number_characters

  puts characters_to_string characters  
end

def characters(number_characters)
  characters = []

  number_characters.times do |index|
    print "#{ index+1 }.-character: "
    character = gets.chomp
    characters << character
  end
  
  characters
end

def characters_to_string(characters)
  "Result: #{characters.join("")}"
end


character_to_string

File: ruby_and_python/3_list/characters_to_string.py
# Convert a list of characters into a string
# Input ['a', 'b', 'c', 'd']
# Output abcd


def character_to_string():
    n = int(input("Number of characters: "))

    characters = get_characters(n)

    print()
    print("Result: "+get_string(characters))

def get_characters(n):
    characters = []

    for i in range(1,n+1):
        character = input("{}.-character: ".format(i))
        characters.append(character)

    return characters

def get_string(characters):
    return "".join(characters)

character_to_string()
